src/section/section_proof_chain.rs

```diff
@@ -92,6 +92,11 @@ impl SectionProofChain {
         1 + self.tail.len()
     }
 
+    /// Index of the last key in the chain.
+    pub(crate) fn last_key_index(&self) -> u64 {
```

Comment:
empty tail (return with 0) will pointing to `head`, right?

Reply:
Yes.